Filter/Proxy/HTTP/DetectServerNames.bambda

```diff
@@ -0,0 +1,51 @@
+/**
+ * Bambda Script to Detect Specific Server Names in HTTP Response
+@author Tur24Tur / BugBountyzip (https://github.com/BugBountyzip)
+  It identifies if the 'Server' header of the HTTP response contains any of the specified server names.
+ * Upon detection, responses are highlighted in red and notes are appended, if enabled.
+ **/
+
+// Convert the list of server names to a set for faster lookup
+Set<String> serverNames = new HashSet<>(Arrays.asList(
```

Comment:
If you use Set.of() then you don't need the intermediary List. This also has the added advantage that it will identify duplicate elements on construction (you have several duplicates that can be removed from the bambda).

Reply:
Thank you Wiener 

The initialization has been changed to Set.of(...), which directly creates an immutable set without the intermediary list. This method is not only more succinct but also more performant. Additionally, Set.of() will throw an IllegalArgumentException if any duplicates are included at the time of set creation, helping to catch errors early.